heynovaio/designstudio
# Build 375 hero restyle

Task: https://app.clickup.com/t/9009201449/BUILD-375

#To Test
- Pull PR branch
- Npm install > npm run build > npm start
- Visit the home and About Us pages (these are the ones that currently have content) 
- Compare Hero sections to Figma, check spacing, functionality, responsive collapsing etc. 
- Note that images aren't as mocked because I couldn't find the images but also I didn't try very hard

Thank you!
